acceptance-tests.md

```diff
@@ -9,25 +9,32 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Request a new account**
     * _As a new user, I should be able to create an account for myself in the NERC._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. A prospective user follows the steps documented in [https://nerc-project.github.io/nerc-docs/get-started/create-a-user-portal-account/](https://nerc-project.github.io/nerc-docs/get-started/create-a-user-portal-account/) to create an account on the NERC.
+    * Acceptance tests:
+        1. For a given username of `computate` for example, [Click here to check that the given username is listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=user.openshift.io~v1~User&name=computate). Note the identities for the given user, specifically one that starts with `github:`, like `github:15114160`.
+        2. For a given users identity, `github:15114160` for example, [Click here to check that the given identity is listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=user.openshift.io~v1~Identity&name=github%3A15114160).
 2. **Remove a user**
     * _As an administrator, I should be able to remove a user, labels association with the user, and user identity from the cluster._
        * _(This step is generally unnecessary, given that a user that doesn’t have authorization to any project, cannot access any allocations, therefore removing a user from allocations is generally enough.)_
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. A NERC admin uses a private runbook to disable/ deactivate a user from KeyCloak.
```

Comment:
Provide a pointer to directions and/or an example.  Note this step needs pass/fail too, if it hasn't already been done.